code/dqc/advanceddtd.q

```diff
@@ -0,0 +1,8 @@
+\d .dqc
+
+advanceddtd:{[tab;adfunc;advars;adsyms]
+  listt:{?[tab;((=;`funct;enlist adfunc);(=;`resultkeys;enlist advars);(=;.Q.pf;x));1b;()]}each (first -2#.Q.PV;(last .Q.PV));
+  t:{value first select from listt[x]`resulttables where sym=adsyms}each (0;1);
```

Comment:
`listt` should be passed as an argument, might also need explicitly defined arguments too:
```suggestion
  t:{[x;y]value first select from x[y]`resulttables where sym=adsyms}[listt]'[0 1];
```